webui/react/src/pages/ExperimentDetails/CodeViewer/CodeViewer.tsx

```diff
@@ -181,7 +181,21 @@ const CodeViewer: React.FC<Props> = ({
         const fileTree = await getExperimentFileTree({ experimentId });
         setIsFetchingTree(false);
 
-        const tree = fileTree.map<TreeNode>((node) => convertV1FileNodeToTreeNode(node));
+        const tree = fileTree
+          .map<TreeNode>((node) => convertV1FileNodeToTreeNode(node))
+          .sort((a, b) => {
```

Comment:
move the sorting into the `convertV1FileNodeToTreeNode`? that way subdirectories are sorted?